community-tutorials/ubuntu-docker-install/01-en.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Install Docker-CE on Ubuntu 20.04
```

Comment:
Please specify more precisely what you are addressing in this tutorial (portainer, docker-compose, uwf) for title, slug, and description